Fakebook.Posts/Fakebook.Posts.Domain/IPostsRepository.cs

```diff
@@ -0,0 +1,14 @@
+using Fakebook.Posts.Domain.Models;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Fakebook.Posts.Domain
+{
+    public interface IPostsRepository : ICollection<Post>, IAsyncEnumerable<Post>
+    {
+        ValueTask<Post> AddAsync(Post post);
+        Task<Post> GetPostByEmail(string email);
+        Task<User> GetUserByEmail(string email);
```

Comment:
These should be in the see-posts user story, also remove the IAsyncEnumerable interface, we don't need it in this story

Reply:
Removed interface not related to create-post.